api/spec/controllers/spree/api/v1/checkouts_controller_spec.rb

```diff
@@ -4,6 +4,36 @@ module Spree
   describe Api::V1::CheckoutsController, type: :controller do
     render_views
 
+    shared_examples_for 'action which loads order using load_order_with_lock' do
+      before do
+        allow(controller).to receive(:load_order).with(true).and_return(true)
+      end
+
+      it 'should invoke load_order_with_lock' do
+        expect(controller).to receive(:load_order_with_lock).exactly(1).times
+      end
+
+      it 'should invoke load_order' do
+        expect(controller).to receive(:load_order).with(true).exactly(1).times.and_return(true)
+      end
+
+      context 'ensure no double_render_error' do
+        before do
+          def controller.load_order(lock = false)
```

Comment:
Unused method argument - `lock`. If it's necessary, use `_` or `_lock` as an argument name to indicate that it won't be used. You can also write as `load_order(*)` if you want the method to accept any arguments but don't care about them.